extensions/src/main/java/org/pentaho/platform/web/http/api/resources/JobScheduleRequest.java

```diff
@@ -101,6 +101,10 @@ public class JobScheduleRequest implements Serializable {
 
   String timeZone;
 
+  protected String runSafeMode;
```

Comment:
@srallapa Please update the copyright header to 2023 for this file and all others.

Reply:
Updated.